src/bilder/images/dagster/templates/docker-compose.yaml.j2

```diff
@@ -72,3 +73,23 @@ services:
       - "traefik.http.middlewares.traefik-forward-auth.forwardauth.address=http://traefik-forward-auth:4181"
       - "traefik.http.middlewares.traefik-forward-auth.forwardauth.authResponseHeaders=X-Forwarded-User"
       - "traefik.http.services.traefik-forward-auth.loadbalancer.server.port=4181"
+  # per node healthcheck container
+  health-check:
+    image: 610119931565.dkr.ecr.us-east-1.amazonaws.com/dockerhub/mitodl/ol-infra-health-checks:latest
+    container_name: health-check
+    ports:
+    - "8097:8907"
+    # volumes:
+    # do I need this? Didn't need to specify when testing locally with ol-data-platform
+    - /var/run/docker.sock:/var/run/docker.sock
+    - /etc/docker/compose:/etc/docker/compose
+    labels:
+      # Explicitly tell Traefik to expose this container
+      - "traefik.enable=true"
+      # Send all health_check host traffic to a router
+      - "traefik.http.routers.healthcheck.rule=Host(`0.0.0.0`)"
```

Comment:
The routing rule that we want to use is a path prefix of `/healthcheck` so that only requests to that path and paths under it get routed to this container.